File: src/Dispatcher.java
/**
 * Клас Диспетчер ДСНС
 */
public class Dispatcher extends Person {
    private String contactInfo;

    public Dispatcher(String name, String contactInfo) {
        super(name, "Диспетчер ДСНС");
        this.contactInfo = contactInfo;
    }

    public void receiveAlert(Event event) {
        System.out.println("Диспетчер " + getName() + " отримав сповіщення про подію типу: " + event.getType());
        System.out.println("Значення: " + event.getMeasure().getValue());
    }
}

File: src/Dispatcher.py
# Клас Диспетчер ДСНС
class Dispatcher(Person):
    def __init__(self, name: str, contact_info: str):
        super().__init__(name, "Диспетчер ДСНС")
        self.contact_info = contact_info

    def receive_alert(self, event):
        print(
            "Диспетчер "
            + self.get_name()
            + " отримав сповіщення про подію типу: "
            + event.get_type()
        )
        print("Значення: " + str(event.get_measure().get_value()))
